torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -471,6 +403,20 @@ def batch_norm_inference(input, weight, bias, mean, var, momentum, eps):
                 output = batch_norm_post_process(output, shape, weight, bias)
                 return g.call_function(target_wrappers.pack_to_tuple, (output,))
 
+            # NOTE(jdh8): Workaround for tenstorrent/tt-metal#12671
+            # Passing a tensor shaped `(N,)` to the kernel results in `(1, N)`.
+            # Reshape the tensor back to get the correct shape.
+            def reshape_1d(code, args=args, kwargs=kwargs):
+                shape = node.meta["val"].size()
+                result = g.call_function(code, args, kwargs)
+                return result if len(shape) > 1 else g.call_function(ttnn.reshape, (result, shape))
```

Comment:
This new implementation only introduces 0&ndash;1 op.  I also rename this tool function to reflect what it does.

Reply:
should this be a call to ttnn.unsqueeze?